### PR TITLE
Profile css code review

### PR DESCRIPTION
**Pull request by pvolnuhi**
[profile-form-FE.zip](https://github.com/ericta-portfolio/CSC308-PolySnaps/files/6530251/profile-form-FE.zip)
Created profile.css and changed up ReactFrom.jsx, ProfileFormPage.jsx, and ImgUpload.jsx. Mainly Cosmetic changes.

### CODE REVIEW

File: frontend/src/profile-form-FE/ReactForm.jsx [hunk 1]
```diff
@@ -171,89 +178,102 @@ export default function ReactForm() {
...
 
       {/* Partying */}
       <h1>Party Favors: {partying}</h1>
       {recreationalsList.map((recreational) => (
         <div>
+          <label className="party-input">
           <input
             onChange={recreationalChange}
-            type="checkbox"
+            type="radio"
```
Comment (jpnoe) [inline frontend/src/profile-form-FE/ReactForm.jsx:271]: I think this should be checkbox, not radio, to allow for multiple selection

File: frontend/src/profile-form-FE/profile.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+.form-input {
+    margin-left:10px;
+}
+
+.prof-input {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```
Comment (jpnoe) [inline frontend/src/profile-form-FE/profile.css:9]: interesting, never seen this type of property/function

**Review by jpnoe — COMMENTED** [review]
Looks good to me! Just the button being switched to radio might be an issue

File: frontend/src/profile-form-FE/ReactForm.jsx (hunk 1 above)
Comment (lalvessiqueira) [reply frontend/src/profile-form-FE/ReactForm.jsx:271]: Correct! It should be a checkbox

**Review by lalvessiqueira — COMMENTED** [review]
Still need to run on local machine to see how it looks!

**Pull request merged**